Source/WebCore/platform/graphics/avfoundation/cf/CDMSessionAVFoundationCF.cpp

```diff
@@ -67,8 +67,9 @@ RefPtr<Uint8Array> CDMSessionAVFoundationCF::generateKeyRequest(const String&, U
     CFDataAppendBytes(certificateData.get(), certificate->data(), certificate->byteLength());
 
     auto assetStr = keyID.utf8();
-    auto assetID = adoptCF(CFDataCreateMutable(kCFAllocatorDefault, assetStr.length()));
-    CFDataAppendBytes(assetID.get(), assetStr.dataAsUInt8Ptr(), assetStr.length());
+    auto assetStrSpan = assetStr.span();
+    auto assetID = adoptCF(CFDataCreateMutable(kCFAllocatorDefault, assetStrSpan.size()));
+    CFDataAppendBytes(assetID.get(), assetStrSpan.data(), assetStrSpan.size());
```

Comment:
Seems like this should use a function that creates a `CFData` from a `span`, one that we already have. Seems highly unlikely this needs to be mutable. Probably same for the certificate above.